examples/pytorch/_tests_requirements.txt

```diff
@@ -25,4 +25,8 @@ torchaudio
 jiwer
 librosa
 evaluate >= 0.2.0
-albumentations
+timm
+# TODO: replace with `albumentations` when version > 1.4.4 released
+git+https://github.com/albumentations-team/albumentations@cc0d5a0c68f9c1d3e3262fd610e1d3bc9807e2fb#egg=albumentations
```

Comment:
Curious, which feature are you leveraging which isn't available in Albumentations yet?

Reply:
I asked them to add box clipping, `clip=True` in BboxParams. If it is not set and box is at least one pixel outside the image the error is raised.